instances.go

```diff
@@ -92,6 +93,17 @@ type InstanceBackup struct {
 	} `json:"schedule,omitempty"`
 }
 
+type instanceDiskEncryption string
```

Comment:
Could we make this public?

```suggestion
type InstanceDiskEncryption string
```

Reply:
We could make it public but then you can pass any string as an `InstanceDiskEncryption` type rather than just the two publically exposed vars. Those are vars rather than const as you can't reference a const.

Reply:
Gotcha, that makes sense! I don't think we have a precedent for private consts like this but it might not be a bad idea to adopt it. 

My main reservation would be if linodego fell out of parity with the API, users would be unable to specify any new consts that don't yet exist in linodego. That might be considered undefined behavior though and is probably a situation we would want to avoid in the first place :thinking:

Reply:
I think it is fine to make it a public type because allowing users to create a custom string to be `InstanceDiskEncryption` won't increase any risk from SDK perspectives. And it has been a common way to handle it in our existing linodego code as well as other SDKs like AWS SDK for Go.

Reply:
Ok that all works for me, I've rebased with the type being public